CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ In particular, calls like `list(G)` will no longer return a list of nodes; use `
 - Passing a `NodeSequence` or `LinkSequence` object to `GraphSAGE` and `HinSAGE` classes is now deprecated and no longer supported [\#498](https://github.com/stellargraph/stellargraph/pull/498).
 Users might need to update their calls of `GraphSAGE` and `HinSAGE` classes by passing `generator` objects instead of `generator.flow()` objects.
 
+- Various methods on `StellarGraph` have been renamed to be more succinct and uniform:
+   - `get_feature_for_nodes` is now `node_features`
+   - `type_for_node` is now `node_type`
```

Comment:
Have you checked if notebooks require updating from this change?

Reply:
potentially in `readthedocs` too?

Reply:
> Have you checked if notebooks require updating from this change?
 
Yep:

```
$ git grep get_feature_for_node 
CHANGELOG.md:   - `get_feature_for_nodes` is now `node_features`
$ git grep type_for_node
CHANGELOG.md:   - `type_for_node` is now `node_type`
stellargraph/core/graph_networkx.py:        type_for_node = {}
stellargraph/core/graph_networkx.py:            type_for_node[n] = self._get_node_type(ndata)
stellargraph/core/graph_networkx.py:            node_types.add(type_for_node[n])
stellargraph/core/graph_networkx.py:                node_features, type_for_node, node_types, dtype
```

The last four are just a local variable.

> potentially in readthedocs too?

I'm not sure I understand? I think the docs are just generated from the code and so there's no separate "read the docs" entity?

Reply:
Thanks

> I'm not sure I understand? I think the docs are just generated from the code and so there's no separate "read the docs" entity?

I was just referring to anything in the `docs` directory that might need to be updated.. but doesn't seem like there's any stellargraph usage code snippets there, so all good

Reply:
Ah, makes sense 👍